Source/WTF/wtf/RangeSet.h

```diff
@@ -155,11 +155,9 @@ class RangeSet final {
             return;
         }
 
-        std::sort(
-            m_ranges.begin(), m_ranges.end(),
-            [&] (const Range& a, const Range& b) -> bool {
-                return a.begin() < b.begin();
-            });
+        std::ranges::sort(m_ranges, [&] (const Range& a, const Range& b) -> bool {
```

Comment:
Please remove the space between the `[&]` and the parameter list. You can also remove the return type here.

```suggestion
        std::ranges::sort(m_ranges, [&](const Range& a, const Range& b) {
```

Reply:
Done.